src/VisualStudio/Core/Def/Implementation/MoveStaticMembers/MoveStaticMembersDialog.xaml

```diff
@@ -0,0 +1,105 @@
+﻿<vs:DialogWindow
+    xmlns:vs="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
Dialog in VS has to meet some standards. (I don't if any folks in UX has mentioned to you)
Please check this and make sure your xaml meets it 
https://devdivdesignguide.azurewebsites.net/windows/dialog-and-window-patterns/dialogs/

Also plz share a screenshot : )